### DISCUSSION TITLE
SD Card FAT32 with MSDOS or GPT Partition Table?

### DISCUSSION THREAD

**Discussion by Dotatay** [roleoroleo/yi-hack-Allwinner-v2#776] [Q&A] [answered]
Hello,
I have ordered the “Yi Pro 2K Home” and would like to prepare the SD Card in advance. I don’t want to install the YI-APP unless it’s absolutely necessary.

I have already read here that the SD Card should be formatted to FAT32. But which partition table should be used: MSDOS, GPT or another one?

**Comment by roleoroleo** [comment, marked as answer]
MBR partition table.

**Reply by Dotatay** [reply]
Thank you so much.
